packages/paste-website/src/pages/foundations/content/product-style-guide/index.mdx

```diff
@@ -40,482 +40,175 @@ export const getStaticProps = async () => {
-- _Month DD, YYYY_ (_May 17, 2020_) &mdash; when the day of the week is unimportant or does not help the user scan.
-- _Month YYYY_ (_May 2020_) &mdash; when only the month is relevant. Useful in subheads to organize chronological lists.
+To create an en-dash:
+- Mac: Option + - (hyphen)
+- PC: Alt + 0150
 
-**Avoid** repeating information. If a history or log organizes its contents using _Month YYYY_ sub-headings, the following line items should use _Day DD_.
-
-- For the subhead _January 2020_, the dates that follow would read _Wednesday 15_, _Tuesday 28_ and so on.
-
-**Do** indicate a date range by using _to_ between values.
-
-- \_January 24 to February 3, 2020
-- _January 24 to 28_
-
-**Do** abbreviate months to their first 3 letters if space is limited.
-
-- _Jan, Feb, Mar_
-
-- Tip: If it's absolutely necessary to cite a specific date numerically, use the [ISO 8601 date format](https://www.w3.org/TR/NOTE-datetime#:~:text=Introduction-,The%20International%20Standard%20for%20the%20representation%20of%20dates%20and%20times,Representation%20of%20dates%20and%20times%22.) YYYY-MM-DD But note that this may not be useful to your user. Always consider which format is quickest to scan and easiest to read aloud.
-
----
-
-#### I/You/We/Us
-
-**Do** refer to the user as _you_ and to Twilio as _we_.
-
-- Tip: If the user has agency or ownership, use ‘I’ or ‘my’.
-  - _If you still can’t connect, contact our support team._
-  - _To access billing information, select My Account._
-  - _I understand._ (This usage is almost exclusively for legal consent.)
-
-<Callout variant="neutral" marginY="space70">
-  <CalloutHeading as="h5">Legal content</CalloutHeading>
-  <CalloutText>
-    Legal content follows different rules to product &mdash; e.g., in Terms of Service, Twilio is usually referenced in
-    the third person. Always check with the legal team if the content regards consent.
-  </CalloutText>
-</Callout>
+Avoid using em-dashes in product copy. They are generally used to offset related information or highlight a break in a sentence. Instead, use short sentences.
 
-**Do** say _we_ instead of _Twilio_
+### I/You/We/Our
 
-- It's us talking, so there's no need to refer to us in the third person.
-- If "Twilio" is included in a product or feature name, that's ok as that's using it as a noun.
+- Speak directly to the user and address them as “you”.
+- Refer to Twilio as “we” instead of “Twilio”, unless there is confusion about who “we” would refer to.
+- It’s OK to use the possessive “our” when referring to Twilio.
+- It’s OK to use “Twilio” as a noun when referring to the product, or as required by legal.
+- Reserve using “I” for legal content (for example “I understand”).
 
----
+### Italics
 
-#### Lists (all)
+Avoid using italic type wherever possible in product content.
+- To draw attention to a word, bold it.
+- If you need to quote a phrase, use quotation marks.
```

Comment:
oooooh can you add a section, too, about avoiding underlines since they're mainly for links?

Reply:
done